doc/man3/OSSL_CRMF_MSG_get0_tmpl.pod

```diff
@@ -34,12 +39,30 @@ OSSL_CRMF_MSG_get_certReqId
  *OSSL_CRMF_CERTID_get0_serialNumber(const OSSL_CRMF_CERTID *cid);
  const X509_NAME *OSSL_CRMF_CERTID_get0_issuer(const OSSL_CRMF_CERTID *cid);
 
+ X509 *OSSL_CRMF_ENCRYPTEDKEY_get1_encCert(const OSSL_CRMF_ENCRYPTEDKEY *ecert,
+                                           EVP_PKEY *pkey, unsigned int flags
+                                           OSSL_LIB_CTX *libctx, const char *propq);
```

Comment:
I would align the argument order with OSSL_CRMF_ENCRYPTEDVALUE_get1_encCert() and probably do the same with OSSL_CRMF_ENCRYPTEDVALUE_decrypt()

Reply:
If I understand correctly, it means argument order should be like -
```
X509 *OSSL_CRMF_ENCRYPTEDKEY_get1_encCert(const OSSL_CRMF_ENCRYPTEDKEY *ecert,
                                          OSSL_LIB_CTX *libctx, const char *propq,
                                          EVP_PKEY *pkey, unsigned int flags);
 unsigned char
 *OSSL_CRMF_ENCRYPTEDVALUE_decrypt(const OSSL_CRMF_ENCRYPTEDVALUE *enc,
                                   OSSL_LIB_CTX *libctx, const char *propq,
                                   EVP_PKEY *pkey, int *outlen);
```

Reply:
Yes, that WFM